### ISSUE TITLE
what went wrong in this code to fine-tune

### ISSUE THREAD

**Issue by deeplxx** [shorxp/tensorlayer-chinese#22]
def train(x_data, y_data):

    x_train, y_train, x_valid, y_valid = split_data(x_data, y_data)

    # train
    #
    sess = tf.InteractiveSession()
    x = tf.placeholder(tf.float32, (None, 227, 227, 3))
    y_ = tf.placeholder(tf.int64, (None, 17))
    net = model_alexnet(x, 17)
    y = net.outputs

    # define cost and metric
    cost = tl.cost.cross_entropy(y, tf.argmax(y_, 1), name='cost')
    correct_predic = tf.equal(tf.argmax(y, 1), tf.argmax(y_, 1))
    acc = tf.reduce_mean(tf.cast(correct_predic, tf.float32))
    # y_op = tf.argmax(tf.nn.softmax(y))
    train_op = tf.train.AdamOptimizer(1e-5).minimize(cost)

    tl.layers.initialize_global_variables(sess)

    # 导入参数 --> 用同一段代码训练过得参数继续训练，提示load success但是fit的时候正确率还是从很
    # 低开始，像是从头开始训练了
    #
    if os.path.isfile('net_params.npz'):
        print('load pre-train params...')
        tl.files.load_and_assign_npz(sess, 'net_params.npz', net)

    tl.utils.fit(sess, net, train_op, cost, x_train, y_train, x, y_,
                 acc, batch_size=32, n_epoch=100, print_freq=10,
                 X_val=x_valid, y_val=y_valid, eval_train=True, tensorboard=True)
    tl.files.save_npz(net.all_params, name='net_params.npz', sess=sess)
    sess.close()

如上注释...

**Comment by deeplxx** [comment]
多看源码多读书...
